docs/how-tos/how-to-setup-colorado-delivery-fee.mdx

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 8
+---
+
+# How to setup Colorado Delivery Fee
+
+Solidus taxation system supports the [Colorado retail delivery fee][colorado-delivery-fee]
+out of the box.
+
+In this how-to, we will see the steps required to configure your store to enable this tax.
+
+## Via Admin Panel
+
+1. Create a new Zone for the state of Colorado.
+2. Create a new TaxRate within that zone, with:
+  - The new Colorado Zone as its zone.
+  - A rate of `0.27`.
+  - `Order Level` as `Tax Rate Level`.
+  - `FlatFee` as its base calculator.
+
+
+![Colorado Delivery Fee Setup](./images/colorado-delivery-fee-setup.gif)
```

Comment:
Not sure about the animated gif, it's ~6Mb. Maybe there are ways to optimize it but still not sure it's worth it.